src/frontend/screens/Quests/index.tsx

```diff
@@ -223,8 +223,8 @@ export function QuestsPage() {
             <SearchBar
               searchText={searchText}
               setSearchText={setSearchText}
-              i18n={{ placeholder: 'Search Quest' }}
-              styles={{ container: { margin: '0px 0px 0px auto' } }}
+              i18n={{ placeholder: 'Search' }}
```

Comment:
Might be good to make this a translation string or reuse an existing translation string